next.config.js

```diff
@@ -7,6 +11,84 @@ const nextConfig = {
   }
 }
 
+// API endpoints that should forwarded to the legacy site
+const legacyAPIPaths = [
+  '/total-ogn',
+  '/total-ogv',
+  '/total-ousd',
+  '/circulating-ogn',
+  '/circulating-ogv',
+  '/social-stats',
+  '/mailing-list/:action*',
+]
+
+// Other pages that should be forwarded to the legacy site
+const legacyPagePaths = [
+  // TODO: Do we still use them?
```

Comment:
For all those the content is totally outdated.
So I'd suggest we just serve the new homepage for all of them.
@micahalcorn Please chime in if you disagree